mithril-client-wasm/Makefile

```diff
@@ -13,7 +13,14 @@ build:
 	wasm-pack build --target web --release --out-name index
 
 test:
-	wasm-pack test --headless --firefox --chrome --node --release
+	pkill -f "mithril-aggregator-fake" || true
+	${CARGO} run -p mithril-aggregator-fake -- -p 8000 &
+	if wasm-pack test --headless --firefox --chrome --node --release; then \
+		pkill -f "mithril-aggregator-fake" || true; \
+	else \
+		pkill -f "mithril-aggregator-fake" || true; \
+		exit 1; \
```

Comment:
Can we avoid the duplication of the `pkill` command here?

Reply:
I'm not sure.
I don't see how to handle the server shutdown in case an error is raised during the tests.

Reply:
With this code, the server won't shutdown in case of a test failure
```
test:
	pkill -f "mithril-aggregator-fake" || true
	${CARGO} run -p mithril-aggregator-fake -- -p 8000 &
	wasm-pack test --headless --firefox --chrome --node --release
	pkill -f "mithril-aggregator-fake" || true
```